File: Sem-4/WT-II/LAB_2/pr3.js
const prompt = require("prompt-sync")();
let n=prompt("Enter number:");

let sum=0;
for(let i=1;i<=n;i++){
    if(i%2==0){
        sum -= i;
    }else{
        sum += i;
    }
}

console.log(`Sum is ${sum}`);

File: Sem-2/Flutter/project_1/lib/LAB_3/Pr3.dart
import 'dart:io';

void main(){
  int n;

  print("Enter number:");
  n=int.parse(stdin.readLineSync()!);
  int count=0;
  for(int i=1;i<=n;i++){
    if(n%i==0)
      count++;
  }
  if(count==2)
    print("Prime number.");
  else
    print("Not prime number.");
}